go/ql/test/query-tests/Security/CWE-079/ReflectedXssGood.go

```diff
@@ -3,16 +3,22 @@ package main
 import (
 	"fmt"
 	"html"
+	"html/template"
 	"net/http"
 )
 
 func serve1() {
+	var template template.Template
+
 	http.HandleFunc("/user", func(w http.ResponseWriter, r *http.Request) {
 		r.ParseForm()
 		username := r.Form.Get("username")
 		if !isValidUsername(username) {
 			// GOOD: a request parameter is escaped before being put into the response
 			fmt.Fprintf(w, "%q is an unknown user", html.EscapeString(username))
+			// GOOD: using html/template escapes values for us
+			template.Execute(w, username)
```

Comment:
We should pass a map or struct to `Execute` or `ExecuteTemplate`, since that's what the method accepts (of course it makes no difference for the test, but a reader could suppose we've misunderstood its usage)

Reply:
As far as I can tell they accept `any`. The examples at the [start of the documentation](https://pkg.go.dev/html/template) just pass in strings (assuming we are talking about the last arguments).